csharp/test/Apache.Arrow.Adbc.Tests/Metadata/GetObjectsParser.cs

```diff
@@ -170,9 +170,17 @@ private static List<AdbcConstraint> ParseConstraints(StructArray constraintsArra
                 c.Name = name.GetString(i);
                 c.Type = type.GetString(i);
 
-                StringArray col_names = column_names.GetSlicedValues(i) as StringArray;
+                StringArray colNames = column_names.GetSlicedValues(i) as StringArray;
```

Comment:
Sorry, should column_names also be named columnNames (I understand that this was before this PR). I'd also suggest fixing the names of other variables in this method.